src/SwedbankPay.Sdk/PaymentInstruments/Card/CardPaymentRequest.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+
+using SwedbankPay.Sdk.PaymentOrders;
+
+namespace SwedbankPay.Sdk.Payments.CardPayments
+{
+    public class CardPaymentRequest
+    {
+        public CardPaymentRequest(Operation operation,
+                              PaymentIntent intent,
+                              CurrencyCode currency,
+                              List<IPrice> prices,
+                              string description,
+                              string userAgent,
+                              CultureInfo language,
+                              IUrls urls,
+                              PayeeInfo payeeInfo,
+                              bool generatePaymentToken = false,
+                              bool generateReccurenceToken = false,
+                              string payerReference = null,
+                              RiskIndicator riskIndicator = null,
+                              Cardholder cardholder = null,
+                              CreditCard creditCard = null,
+                              Dictionary<string, object> metadata = null,
+                              string paymentToken = null)
+        {
+            this.Payment = new CardPaymentRequestObject(operation, intent, currency, prices, description, payerReference, generatePaymentToken,
+                                               generateReccurenceToken, userAgent, language, urls, payeeInfo, riskIndicator, cardholder,
+                                               creditCard, metadata, paymentToken);
+        }
+
+
+        public CardPaymentRequestObject Payment;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)